scripts/dpla_ingest_update.py

```
@@ -52,28 +52,22 @@ def get_json(url, params=None):
-        gold[hub] = date
-
-g30 = dict(sorted(g30.items(), key=lambda x: x[1], reverse=True))
-g90 = dict(sorted(g90.items(), key=lambda x: x[1], reverse=True))
-g365 = dict(sorted(g365.items(), key=lambda x: x[1], reverse=True))
-gold = dict(sorted(gold.items(), key=lambda x: x[1], reverse=True))
-
-print(f"30 days:  {len(g30)} hubs")
-print(f"90 days:  {len(g90)} hubs")
-print(f"365 days: {len(g365)} hubs")
-print(f"Older:    {len(gold)} hubs")
+    for group, (days, _) in zip(groups, BUCKETS):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)